sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -344,49 +283,42 @@ export interface CustomAnalyzer {
 
 // @public
 export interface CustomEntity {
-    accentSensitive?: boolean;
-    aliases?: CustomEntityAlias[];
-    caseSensitive?: boolean;
-    defaultAccentSensitive?: boolean;
-    defaultCaseSensitive?: boolean;
-    defaultFuzzyEditDistance?: number;
-    description?: string;
-    fuzzyEditDistance?: number;
-    id?: string;
+    accentSensitive?: boolean | null;
```

Comment:
feels a bit odd to support `null` | `undefined`, but is this intentional?

Reply:
[This APIView](https://apiview.dev/Assemblies/Review/9cb5a2896ebd4eefa96595b00c9a3cd4/fab0242f58ee450b80fb859da8a77698?diffRevisionId=33d00ebc99004eb984cf7d558ceff3fe&doc=False&diffOnly=True) compares the 11.3.1 release tag with the most recent commit in this branch (9ea4d7c): 
The nullable fields are GA surface. The beta surface has these marked as optional fields (by the `openapi-type: data-plane` field in the Swagger), and I'm thinking it'd be best to roll that into GA surface with the next major version.